### PR TITLE
Fixed logic in many methods (part 2).

### PR DESCRIPTION
**Pull request by SidRoberts**
Hello!

* Type: code quality
* Link to issue:

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [-] I wrote some tests for this PR
- [-] I updated the CHANGELOG

Small description of change:

Thanks

### CODE REVIEW

**Comment by SidRoberts** [comment]
Another commit is incoming. :stuck_out_tongue_closed_eyes:

**Review by sergeyklay — DISMISSED** [review]

File: phalcon/Http/Request.zep [hunk 1]
```diff
@@ -206,13 +205,13 @@ class Request implements RequestInterface, InjectionAwareInterface
...
-            /**
-             * @see https://bugs.php.net/bug.php?id=66606
-             */
-            if fetch contentType, _SERVER["HTTP_CONTENT_TYPE"] {
-                return contentType;
-            }
+        }
+
+        /**
+         * @see https://bugs.php.net/bug.php?id=66606
```
Comment (Jeckerson) [inline phalcon/Http/Request.zep]: Current bug was fixed in `5.5.8` which Phalcon 4 doesn't support...

Comment (SidRoberts) [reply phalcon/Http/Request.zep]: Thanks for the heads up. I've removed it in 7c81838

**Review by Jeckerson — APPROVED** [review]

**Comment by SidRoberts** [comment]
I've finished with this now so it can be merged whenever.

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @SidRoberts

**Pull request merged**